ErikSteLarsen/Gruppe2EiT
# Add frontend client outline

### SUMMARY
Adds a `web/client` folder where the React client lives.

### HOW TO VERIFY
From the `web/client` folder:
1. Run `npm install`
2. Run `npm start`

This start the React development server and open the current version in your browser. It reloads on file change. See `web/client/README.md` for more details.

### NOTES
* Contains a to-be-implemented `WeightMonitorContainer`.

Closes #69
